### ISSUE TITLE
[BUG]: Unable to Move Window on M1 MacBook Pro (v12.5.1)

### ISSUE THREAD

**Issue by spyroskotsakis** [Mintplex-Labs/anything-llm#1614]
### How are you running AnythingLLM?

AnythingLLM desktop app

### What happened?

Hello,

I'm sorry if this has been reported before. For some reason, the window can't be moved within the screen. I'm using an M1 MacBook Pro v 12.5.1. 
I can resize the window, so for now, the only way to move it around on my screen is by resizing it.

Thanks a lot.

**Comment by timothycarambat** [comment]
Closing as unable to replicate. Certainly can do this right now. Are you clicking on the top title bar of the window? The only place you cannot click and drag is where the stoplight controls are.

**Comment by nyo-at-work** [comment]
@timothycarambat , 
I experienced the same issue yesterday. When I launched the app for the first time, the 'Getting Started' screen appeared, and I couldn't move the window by clicking anywhere, including the title bar. Although I could minimize, maximize, and resize the window, I couldn't drag it to a different location. I feel like the app was not responsive at that point. The only place I could click was on the 'Getting Started' button, and I had to follow through the setup process and configure an LLM. Only after completing these steps did the window behavior return to normal, and I could move it around and become like any other app. I'm using a MacBook Pro M1 Pro with macOS 14.5 and AnythingLLM Desktop version 1.5.8. Thanks for the great work!
